src/components/ActivityFeed/ActivityFeed.js

```diff
@@ -152,7 +153,7 @@ const resolveTransitionMessage = (
       const userHasLeftAReview = hasUserLeftAReviewFirst(ownRole, transaction);
 
       const reviewLink =
-        reviewPeriodJustStarted || !(reviewPeriodIsOver || userHasLeftAReview) ? (
+        reviewPeriodJustStarted && !(reviewPeriodIsOver || userHasLeftAReview) ? (
```

Comment:
I don’t understand this change.

If the user is reviewing as a second person:
false && !(false || false)
false && true 
false

It’s probably better to refactor this a bit:

```js
const waitingForMissingSecondReview = !(reviewPeriodIsOver || userHasLeftAReview);
const reviewLink = reviewPeriodJustStarted || waitingForMissingSecondReview ? (
```

Reply:
This part is only for the first review. The second review is handled in a different case (but it seems there was a mistake in the code comment). 

So actually checking `!(reviewPeriodIsOver || userHasLeftAReview)` is not needed here at all because it's enough to check that the current state is "delivered".